solidity/contracts/tbtc/Vault.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity 0.8.4;
+
+import "../bank/Bank.sol";
+import "../token/TBTC.sol";
+
+/// @title TBTC application vault
+/// @notice TBTC is a fully Bitcoin-backed ERC-20 token pegged to the price of
+///         Bitcoin. It facilitates Bitcoin holders to act on the Ethereum
```

Comment:
There is 1:1 ratio between Bitcoin balance in the Bank and TBTC minted but there is no 1:1 peg between Bitcoin price and TBTC price. Currently, TBTCv1 trades at a premium because of the minting cost and the same could happen to TBTCv2. I think we should not commit to 1:1 ratio.

Reply:
Ok, got it! Thanks for the clarification.